code/modules/antagonists/cult/cult.dm

```diff
@@ -215,7 +234,52 @@
 	var/cult_vote_called = FALSE
 	var/mob/living/cult_master
 	var/reckoning_complete = FALSE
-
+	var/cult_risen = FALSE
+	var/cult_ascendent = FALSE
+	
+/datum/team/cult/proc/check_size()
+	if(cult_ascendent)
+		return
+	var/alive = 1
+	var/cultplayers = 0
+	for(var/I in GLOB.player_list)
+		var/mob/M = I
+		if(M.stat != DEAD)
+			if(iscultist(M))
+				++cultplayers
+			else
+				++alive
+	var/ratio = cultplayers/alive
```

Comment:
Technically alive can be zero here.

Reply:
This compiles, but doesn't that divide by zero? Or does BYOND just not care if it's 0/0?

Reply:
Not sure what you mean but this is not compile time value, so byond has no idea if this is division by zero.

Reply:
If there's no cultyplayers or alive you can just return since it will won't need update then.